### PR TITLE
correct the customerSpecifiedDeliveryDateTime tag in documentation

### PR DESCRIPTION
**Pull request by AnshikaKoul**
New customers integrating with booking API for OX reported this problem.

https://github.com/bring/booking/blob/master/webapp/src/main/resources/schemas/booking-v1.xsd#L80

Screenshot 2021-03-12 at 4 25 04 PM

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/mrb13Jfa](https://trello.com/c/mrb13Jfa)

File: content/api/booking/schemas/booking-v1.xsd [hunk 1]
```diff
@@ -76,7 +76,7 @@
     <xs:complexType name="ConsignmentRequestType">
         <xs:sequence>
             <xs:element name="shippingDateTime" type="xs:dateTime"/>
-            <xs:element name="customerSpecifiedDeliveryDate" type="xs:date" minOccurs="0"/>
+            <xs:element name="customerSpecifiedDeliveryDateTime" type="xs:date" minOccurs="0"/>
```
Comment (tomgag) [inline content/api/booking/schemas/booking-v1.xsd:79]: Late to the party, but should this be `xs:dateTime`?

File: content/api/booking/examples/basicRequestNew.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <consignments>
     <consignment correlationId="INTERNAL-123456">
       <shippingDateTime>2020-05-05T21:31:33.007+02:00</shippingDateTime>
-      <customerSpecifiedDeliveryDate>2020-05-11</customerSpecifiedDeliveryDate>
+      <customerSpecifiedDeliveryDateTime>2020-05-11</customerSpecifiedDeliveryDateTime>
```
Comment (tomgag) [inline content/api/booking/examples/basicRequestNew.xml:7]: Should this be `2020-05-11T00:00:00` ?

File: content/api/booking/examples/basicRequestNew.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "consignments": [
     {
       "shippingDateTime": 1588678607000,
-      "customerSpecifiedDeliveryDate": "2020-05-11",
+      "customerSpecifiedDeliveryDateTime": "2020-05-11",
```
Comment (tomgag) [inline content/api/booking/examples/basicRequestNew.json:7]: Not sure whether we support this format for JSON dateTime?

File: content/api/booking/examples/booking_soap_request.xml [hunk 4]
```diff
@@ -7,7 +7,7 @@
       <consignments>
         <consignment correlationId="INTERNAL-123456">
           <shippingDateTime>2020-05-05T21:31:33.007+02:00</shippingDateTime>
-          <customerSpecifiedDeliveryDate>2020-05-11</customerSpecifiedDeliveryDate>
+          <customerSpecifiedDeliveryDateTime>2020-05-11</customerSpecifiedDeliveryDateTime>
```
Comment (tomgag) [inline content/api/booking/examples/booking_soap_request.xml:10]: same here

**Pull request merged**